presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -750,6 +753,27 @@ std::string PrestoServer::connectorOperation(
   }
 }
 
+std::string PrestoServer::systemConfigOperation(
+    const ServerOperation& op,
+    proxygen::HTTPMessage* message) {
+  switch (op.action) {
+    case ServerOperation::Action::kSetProperty: {
+      const auto name = message->getQueryParam("name");
+      const auto value = message->getQueryParam("value");
```

Comment:
What type of validation do you have in mind?
Just being non-empty?
Because validation for specific settings - that is likely not feasible for all props.
Keep in mind that this is only for dev clusters, not for production.

Reply:
No like, if the user does not provide "name" param or "value" param. The message should indicate the accurate error instead of letting SystemConfig handle it and throw a confusing message.

Reply:
Maybe can also add in test about this scenario.

Reply:
How can we add tests with http commands, do we have one already?